backend/user.js

```diff
@@ -569,22 +572,41 @@ router.put('/patients/:id/toggle-notifications', authenticateToken, async (req,
       return res.status(404).json({ error: 'Patient not found' });
     }
 
+    const { conditions, totalWeight } = await checkNotificationConditions(patientId);
+
+    if (totalWeight >= 5) {
```

Comment:
do we also want to store these weight cutoff values (1 and 5) as variables? the variable names may help reviewers have a better understanding of what they represent and will make changing the values more convenient down the line (if you decide to do that).

Reply:
on it!